src/main/java/de/tum/in/www1/artemis/service/compass/umlmodel/classdiagram/UMLAttribute.java

```diff
@@ -99,7 +99,7 @@ private boolean parentsSimilar(UMLAttribute referenceAttribute) {
 
     @Override
     public String toString() {
-        return "Attribute " + name + (attributeType != null && !"".equals(attributeType) ? ": " + attributeType : "") + " in class " + parentElement.getName();
+        return "Attribute " + name + (attributeType != null && !attributeType.isEmpty() ? ": " + attributeType : "") + " in class " + parentElement.getName();
```

Comment:
Probably ok, but might there be any possibility that attributeType is null?
Especially for debugging it might be better to have null safety here

Reply:
`attributeType != null && !attributeType.isEmpty()`